CIP5/README.md

```diff
@@ -23,31 +23,58 @@ Therefore, we can leverage bech32 for binary data encoding, with a set of common
-| `stake_xsk`  | Stake address extended signing key            |
-| `stake`      | Mainnet stake address                         |
-| `stake_test` | Testnet stake address                         |
-| `pool`       | Pool operator verification key hash (pool ID) |
-| `pool_vk`    | Pool operator verification key                |
-| `pool_sk`    | Pool operator signing key                     |
-| `kes_vk`     | KES verification key                          |
-| `kes_vkh`    | KES verification key hash                     |
-| `kes_sk`     | KES signing key                               |
-| `vrf_vk`     | VRF verification key                          |
-| `vrf_vkh`    | VRF verification key hash                     |
-| `vrf_sk`     | VRF signing key                               |
+#### Keys
+
+| Prefix       | Meaning                                 | Contents                           |
+| ---          | ---                                     | ---                                |
+| `acct_sk`    | CIP-1852's Account private key          | Ed25519 private key                |
+| `acct_vk`    | CIP-1852's Account public key           | Ed25519 public key                 |
+| `acct_xsk`   | CIP-1852's extended Account private key | Ed25519-bip32 extended private key |
+| `acct_xvk`   | CIP-1852's extended Account public key  | Ed25519 public key with chain code |
+| `addr_sk`    | Address signing key                     | Ed25519 private key                |
+| `addr_vk`    | Address verification key                | Ed25519 public key                 |
+| `addr_xsk`   | Address extended signing key            | Ed25519-bip32 extended private key |
+| `addr_xvk`   | Address extended verification key       | Ed25519 public key with chain code |
+| `kes_sk`     | KES signing key                         | KES signing key                    |
+| `kes_vk`     | KES verification key                    | KES verification key               |
+| `pool_sk`    | Pool operator signing key               | Ed25519 private key                |
+| `pool_vk`    | Pool operator verification key          | Ed25519 public key                 |
+| `root_sk`    | CIP-1852's root private key             | Ed25519 private key                |
+| `root_vk`    | CIP-1852's root public key              | Ed25519 public key                 |
+| `root_xsk`   | CIP-1852's extended root private key    | Ed25519-bip32 extended private key |
+| `root_xvk`   | CIP-1852's extended root public key     | Ed25519 public key with chain code |
+| `script_sk`  | Script signing key                      | Ed25519 private key                |
+| `script_vk`  | Script verification key                 | Ed25519 public key                 |
+| `script_xsk` | Extended script signing key             | Ed25519-bip32 extended private key |
+| `script_xvk` | Extended script verification key        | Ed25519 public key with chain code |
```

Comment:
For consistency, perhaps we should also have "about" sections for each of these new prefixes under the "rationale" heading.